info-compiler/src/test/java/com/google/sps/NewsContentExtractorTest.java

```diff
@@ -78,8 +79,8 @@ public void extractContentFromHtml_regularTextDoc() throws IOException, SAXExcep
     Optional<NewsArticle> potentialNewsArticle =
         newsContentExtractor.extractContentFromHtml(boilerpipeHandler, metadata, webpageStream,
                                                     URL);
-    Assert.assertTrue(potentialNewsArticle.isPresent());
-    Assert.assertEquals(new NewsArticle(TITLE, URL, CONTENT), potentialNewsArticle.get());
+    assertThat(potentialNewsArticle).isPresent();
+    assertThat(new NewsArticle(TITLE, URL, CONTENT)).isEqualTo(potentialNewsArticle.get());
```

Comment:
Can directly test optional value
https://truth.dev/api/1.0.1/com/google/common/truth/OptionalSubject.html
```suggestion
    assertThat(potentialNewsArticle).hasValue(new NewsArticle(TITLE, URL, CONTENT));
```